webrender/src/internal_types.rs

```diff
@@ -316,21 +316,25 @@ pub struct PackedTexel {
 impl PackedTexel {
     pub fn high_bytes(color: &ColorF) -> PackedTexel {
         PackedTexel {
-            b: (0.5 + color.b * COLOR_FLOAT_TO_FIXED).floor() as u8,
-            g: (0.5 + color.g * COLOR_FLOAT_TO_FIXED).floor() as u8,
-            r: (0.5 + color.r * COLOR_FLOAT_TO_FIXED).floor() as u8,
-            a: (0.5 + color.a * COLOR_FLOAT_TO_FIXED).floor() as u8,
+            b: Self::normalized_f32_to_wrapped_u8(color.b, COLOR_FLOAT_TO_FIXED),
```

Comment:
what I meant by a "helper function" is something like `fn extract_bytes(color: &ColorF, multiplier: f32) -> PackedTexel`